tests/integration/source_code/solacc.py

```diff
@@ -44,16 +45,27 @@ def clone_all():
         run_command(f'git clone {url} {dst}')
 
 
-def lint_all():
+def lint_all(file_to_lint: str | None):
     # pylint: disable=too-many-nested-blocks
     ws = WorkspaceClient(host='...', token='...')
     ctx = LocalCheckoutContext(ws).replace(
         linter_context_factory=lambda session_state: LinterContext(MigrationIndex([]), session_state)
     )
     parseable = 0
     missing_imports = 0
-    all_files = list(dist.glob('**/*.py'))
+    all_files = list(dist.glob('**/*.py')) if file_to_lint is None else [Path(dist, file_to_lint)]
+    if file_to_lint is None:
+        unparsed = Path(Path(__file__).parent, "solacc-unparsed.txt")
+        if unparsed.exists():
+            os.remove(unparsed)
```

Comment:
Instead of removing the file and then adding a line at a time as things run, what do you think about:
1. Starting with an empty list;
2. Filling the list as things run;
3. Replacing the file once at the end (with `.open("wt"`).

Reply:
I think it's clearer to remove the file upfront